### PR TITLE
netlib-lapack: fix cmake error with FJ compiler (like #12392).

### PR DESCRIPTION
**Pull request by h-murai**
The error discussed in #12392 is found also for FJ compiler. This patch is the same as for #12392 and fixes the error.

### CODE REVIEW

**Comment by bernhardkaindl** [comment]
@h-murai This needs a rebase to current develop (sorry for the delay) and you should request review from someone who changed this package in the past, if possible the maintainer of that package.

You can request the review from the "Reviewers" field on the top right corner in github's PR view.

File: var/spack/repos/builtin/packages/netlib-lapack/package.py [hunk 1]
```diff
@@ -49,6 +49,8 @@ class NetlibLapack(CMakePackage):
     patch('ibm-xl-3.9.1.patch', when='@3.9.1 %xl')
     patch('ibm-xl-3.9.1.patch', when='@3.9.1 %xl_r')
 
+    patch('fj.patch', when='@3.7: %fj')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/netlib-lapack/package.py:52]: Can you add a comment above the directive explaining briefly what the patch does and where does it come from?

**Review by alalazo — CHANGES_REQUESTED** [review]
Apologies for the long delay in review! I missed this and am going through old PRs today.

File: var/spack/repos/builtin/packages/netlib-lapack/package.py (hunk 1 above)
Comment (alalazo) [reply var/spack/repos/builtin/packages/netlib-lapack/package.py:52]: @h-murai

**Comment by alalazo** [comment]
Closing, since there has been no activity in a while. Feel free to reopen if you want to continue work on this.

**Pull request closed without merging**